src/nipplejs.js

```diff
@@ -135,17 +135,13 @@ var Nipple = function (options) {
 // Basic event system.
 Nipple.prototype.on = function (type, cb) {
     var self = this;
-    var types = type.split(/[ ,]/g);
+    var types = type.split(/[ ,]+/g);
 
-    if (types.length > 1) {
-        for (var i = 0, max = types.length; i < max; i += 1) {
-            self.on(types[i], cb);
-        }
-        return self;
+    for (var i = 0; i < types.length; ++i) {
+        var type = types[i];
```

Comment:
Can you declare this out of the loop please, only assign it inside?

```javascript
var type;
for (var i = 0; i < types.length; i += 1) {
    type = types[i];
    [...]
}
```
also use the `i += 1` form, for clarity :sunglasses: